From: Takashi Iwai <tiwai@suse.de>
Subject: Fix kABI breakage for iwl_fw_runtime_ops change
Patch-mainline: Never, SLE15 only
References: bsc#1051510

The patch patches.drivers/iwlwifi-avoid-collecting-firmware-dump-if-not-loaded
breaks kABI due to the addition of fw_running to iw_fw_runtime_ops.
Let's hide it.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/net/wireless/intel/iwlwifi/fw/runtime.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/wireless/intel/iwlwifi/fw/runtime.h
+++ b/drivers/net/wireless/intel/iwlwifi/fw/runtime.h
@@ -70,7 +70,9 @@
 struct iwl_fw_runtime_ops {
 	int (*dump_start)(void *ctx);
 	void (*dump_end)(void *ctx);
+#ifndef __GENKSYMS__
 	bool (*fw_running)(void *ctx);
+#endif
 };
 
 #define MAX_NUM_LMAC 2
